.github/workflows/update_chains_preconfigured.yaml

```diff
@@ -25,8 +25,6 @@ jobs:
 
       - name: ⚙️ Install dependencies
         run: make init
-      - name: Clear old files
```

Comment:
How will we delete outdated configurations?

Reply:
hmm, maybe it could be semi-automatic and we can accept changes with some amendments (adding absent marks)?